tools/cli/commands.go

```diff
@@ -369,7 +364,7 @@ func UpdateDomain(c *cli.Context) {
 		if c.IsSet(FlagEmitMetric) {
 			emitMetric, err = strconv.ParseBool(c.String(FlagEmitMetric))
 			if err != nil {
-				ErrorAndExit("Update Domain failed", err)
+				ErrorAndExit(FlagEmitMetric+"format is invalid.", err)
```

Comment:
missing space

Reply:
Updated. Will send out a new PR about the change.